web/client/observables/wfs.js

```diff
@@ -195,7 +195,7 @@ export const getJSONFeature = (searchUrl, filterObj, options = {}) => {
     return Rx.Observable.defer(() =>
         axios.post(queryString, data, {
             timeout: 60000,
-            headers: { 'Accept': 'application/json', 'Content-Type': 'application/json' }
+            headers: { 'Accept': 'application/json', 'Content-Type': `application/xml` }
```

Comment:
why this?

Reply:
Here there was a mistake. Content is an XML but content-type in the heder was "application/json". This was not causing any issue, because probably the content-type is not evaluated by GeoServer for the POST request, assuming it is the standard XML as for the protool. 
Anyway in the updated axios interpret Content-Type of type "json" as a side effect. The body is forwarded as a json string so the final body will be `"<xml ...>"` with `"` at start and end, that is invalid for GeoServer.